zap/src/main/java/org/zaproxy/zap/control/AddOnClassLoader.java

```diff
@@ -320,16 +328,12 @@ protected Class<?> findClass(String name) throws ClassNotFoundException {
             }
         }
 
-        try {
```

Comment:
I think we should keep this the default behaviour (classes of the add-on should have priority over the ones in the dependencies) and allow to change it when needed.